chef/cookbooks/crowbar/templates/default/apache.conf.erb

```diff
@@ -44,6 +44,7 @@
     <% unless @ssl_crt_chain_file.nil? or @ssl_crt_chain_file.empty? %>
     SSLCertificateChainFile <%= @ssl_crt_chain_file %>
     <% end %>
+    RequestHeader set X_FORWARDED_PROTO 'https'
```

Comment:
According to [1] there is a standardized version of this header called `forwarded`. I have no idea how standardized that really is in terms of browser support, but we could add the 'standard' header just to make doubly sure that the browser gets what we are asking it to do:
```
Forwarded: proto=https
```

[1] https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/X-Forwarded-Proto

Reply:
Good catch - it looks like the RFC for that is only three years old https://tools.ietf.org/html/rfc7239 so I'm guessing it's not universal yet but would be good for future-proofing

Reply:
done